src/mmw/js/src/compare/views.js

```diff
@@ -255,6 +258,88 @@ var InputsView = Marionette.LayoutView.extend({
         this.ui.tableButton.addClass('active');
         this.model.set({ mode: models.constants.TABLE });
     },
+
+    downloadCSV: function() {
+        var aoi = App.currentProject.get('area_of_interest'),
+            aoiVolumeModel = new tr55Models.AoiVolumeModel({ areaOfInterest: aoi }),
+            csvHeadings = [['scenario_name', 'runoff', 'evapotranspiration', 'infiltration',
+            'tss_load', 'tss_runoff', 'tss_loading_rate', 'tn_load', 'tn_runoff', 'tn_loading_rate',
+            'tp_load', 'tp_runoff', 'tp_loading_rate']],
+            csvData = this.model.get('scenarios')
+                .map(function(scenario) {
+                    var scenarioName = scenario.get('name'),
+                        result = scenario
+                            .get('results')
+                            .findWhere({ name: 'runoff' })
+                            .get('result'),
+                        runoff,
+                        quality,
+                        tss,
+                        tn,
+                        tp;
+
+                    switch (scenarioName) {
```

Comment:
If the above comment doesn't happen, prefer not switching on name, and instead using the boolean attributes `is_current_conditions` and `is_precolumbian`.

Reply:
Decided to keep this code but adjust the checks to be on `is_current_conditions` and `is_pre_columbian` rather than checking on the scenario names.